Jetcaster/glancewidget/src/main/java/com/example/jetcaster/glancewidget/JetcasterAppWidget.kt

```diff
@@ -209,22 +224,44 @@ private fun AlbumArt(
 @Composable
 fun PodcastText(title: String, subtitle: String, modifier: GlanceModifier = GlanceModifier) {
     val fgColor = GlanceTheme.colors.onPrimaryContainer
-    Column(modifier) {
-        Text(
-            text = title,
-            style = TextStyle(fontSize = 16.sp, fontWeight = FontWeight.Medium, color = fgColor),
-            maxLines = 2,
-        )
-        Text(
-            text = subtitle,
-            style = TextStyle(fontSize = 14.sp, color = fgColor),
-            maxLines = 2,
-        )
+    val size = LocalSize.current
+    when {
+        size.height >= Sizes.short -> Column(modifier) {
+            Text(
+                text = title,
+                style = TextStyle(
+                    fontSize = 16.sp,
+                    fontWeight = FontWeight.Medium,
+                    color = fgColor
+                ),
+                maxLines = 2,
+            )
+            Text(
+                text = subtitle,
+                style = TextStyle(fontSize = 14.sp, color = fgColor),
+                maxLines = 2,
+            )
+        }
+        else -> Column(modifier) {
+            Text(
+                text = title,
+                style = TextStyle(
+                    fontSize = 12.sp,
+                    fontWeight = FontWeight.Medium,
+                    color = fgColor
+                ),
```

Comment:
should this be getting the style from the theme itself?

Reply:
The whole TextStyle though I mean, fontSize, weight etc.

Reply:
Perhaps, but I feel like that refactor is outside of the scope of this PR. The goal was to add widget previews.

Reply:
Thinking more, yes it could, no I don't think we should.

We'd need to build a bunch of XML resources, and that's not the pattern we're tyring to demonstrate with Glance widgets. See : https://github.com/android/platform-samples/blob/main/samples/user-interface/appwidgets/src/main/java/com/example/platform/ui/appwidgets/glance/layout/collections/layout/ImageTextListLayout.kt#L383  for an example